gulpfile.babel.js

```diff
@@ -41,15 +42,33 @@ gulp.task('mocha', (cb) => {
     });
 });
 
-const DOCKER_RUN = 'docker run -d -p 8080:8080 --name eureka netflixoss/eureka:1.1.147';
+function waitForEureka (cb) {
+  request.get({ url: `http://localhost:${DOCKER_PORT}/eureka` }, (err, res, body) => {
```

Comment:
This breaks our `no-use-before-define` rules, so this function should be declared after the constants.